api/query/search_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build small
```

Comment:
Could you remove this line and the same one added to api/query/search.go please? This syntax is only valid for go 1.17+ https://pkg.go.dev/go/build/constraint

Reply:
Yes! This was automatically added by my editor - I'll remove it.